docs/infrastructure/workers/dynamic-worker-pools.md

```diff
@@ -28,56 +28,82 @@ Customers cannot add new workers nor make changes to the configuration of the wo
 
 ### Types of Dynamic Worker
 
-Each dynamic worker pool can specify the worker image used. As of July 2020, `WindowsDefault` is the default, which is Windows Server Core 2016. `Windows2019` (Windows Server Core 2019) worker images are rolling out. After the rollout is complete, Windows Server Core 2016 will be marked as deprecated.
+Each dynamic worker pool can specify the worker image used. As of August 2020, Windows Server Core 2016 is the default. Windows Server Core 2019 and Ubuntu Server 18.04 worker images are also available.
 
-Editing a dynamic worker pool allows you to modify the image used.
+Editing a dynamic worker pool allows you to modify the image used. You can also setup a new worker pool to have some workers using different target images.
+
+The available Worker Images lists specific operating versions (eg `Windows Server Core 2016`, `Windows Server Core 2019`) but also generic "default" options such as `Windows (default)`. Choosing the default option means that your worker will get the latest stable worker image released. This is a good option to choose if you're running a basic script that doesn't have any real dependencies.
+
+If you're writing a script that relies on a specific version of tooling (eg helm), then we recommend choosing a specfic worker image instead of the "default" options, to prevent worker image upgrades from impacting your deployments.
```

Comment:
```suggestion
If you're writing a script that relies on a specific version of tooling (e.g., helm), then we recommend choosing a specific worker image, instead of the "default" options, to prevent worker image upgrades from impacting your deployments.
```